httputils/httphelpers.go

```diff
@@ -179,12 +179,12 @@ func splitSlice2Chunks[T any](slice []T, maxSize int, chunks chan<- []T, wg *syn
 		//slice is bigger than max size
 		//split the slice to slices smaller than max size
 		index := 0
-		for i := 1; i < len(slice); i++ {
-			jsonSize = JSONSize(slice[index:i])
+		for i, _ := range slice {
```

Comment:
@dwertent it's not obvious from the change, but I was missing one more pass, so if the slice was just 1 element too big, it would loop forever